samples/cooperative-cancellation/sample.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Cooperative cancellation
+summary: Stopping both the endpoint and a long running handler with a cancellation token
+reviewed: 2021-08-31
+component: Core
+---
+
+
+Run the solution. One console application starts up, `Server`.
```

Comment:
Skimming through the content I originally read "Once the console application starts up" and then got confused by the sentence. Changing it makes it less likely for the same skimming mistake to happen to others.

```suggestion
Run the solution. A single console application starts up: `Server`.
```